server/cla.go

```diff
@@ -54,80 +59,108 @@ func (s *Server) checkCLA(pr *model.PullRequest) {
 		return
 	}
 
+	body, errCSV := s.getCSV()
+	if errCSV != nil {
+		return
+	}
+
+	if !isNameInCLAList(strings.Split(string(body), "\n"), username) {
+		comments, _, err := s.GithubClient.Issues.ListComments(context.TODO(), pr.RepoOwner, pr.RepoName, pr.Number, nil)
```

Comment:
I'm aware that this is old code, but let me point out that this doesn't  work if there are a lot of commits because paging is not implemented. Maybe a `100` as `PerPage` is fine for now, if implementing paging is to much.

Reply:
Added pagination.